adaptivefiltering/apps.py

```diff
@@ -515,24 +515,116 @@ def _finalize(_):
     return pipeline_proxy
 
 
+def setup_rasterize_side_panel(dataset):
+    # Get a widget for rasterization
+    raster_schema = copy.deepcopy(load_schema("rasterize.json"))
+
+    # We drop classification, because we add this as a specialized widget
+    raster_schema["properties"].pop("classification")
+
+    rasterization_widget_form = ipywidgets_jsonschema.Form(
+        raster_schema, vertically_place_labels=True
+    )
+    rasterization_widget = rasterization_widget_form.widget
+    rasterization_widget.layout = fullwidth
+
+    # Get a widget that allows configuration of the visualization method
+    schema = load_schema("visualization.json")
+
+    form = ipywidgets_jsonschema.Form(schema, vertically_place_labels=True)
+    formwidget = form.widget
+    formwidget.layout = fullwidth
+
+    # Create the classification widget
+    classification = ipywidgets.Box([classification_widget([dataset])])
+    classification.layout = fullwidth
+
+    widged_list = [rasterization_widget, formwidget, classification]
+    form_list = [rasterization_widget_form, form]
+
+    return widged_list, form_list
+
+
 def create_segmentation(dataset):
     """The Jupyter UI to create a segmentation object from scratch.
 
     The use of this UI will soon be described in detail.
     """
-
+    # create instance of dataset with srs of "EPSG:3857",
+    # this ensures that the slope and hillshade overlay fit the map projection.
+    dataset = reproject_dataset(dataset, "EPSG:3857")
     # Create the necessary widgets
+
     map_ = Map(dataset=dataset)
     map_widget = map_.show()
     finalize = ipywidgets.Button(description="Finalize")
 
+    # add control surface for rasterization overlay.
+    if not isinstance(dataset, DigitalSurfaceModel):
+        dataset = dataset.rasterize()
```

Comment:
This one slightly bothers me. It calculates a rasterization that is not even used if the resolution parameter is changed by the user.